src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```
@@ -2220,3 +2220,11 @@ services:
     public: false
     arguments:
       $locales: "@=service('prestashop.adapter.legacy.context').getLanguages()"
+
+  PrestaShopBundle\Form\Admin\Type\ApeType:
+    class: 'PrestaShopBundle\Form\Admin\Type\ApeType'
+    public: true
```

Do you really need to make it public ?

```suggestion
  PrestaShopBundle\Form\Admin\Type\ApeType:
```